docs/userguide/src/site/pages/thredds/ThreddsConfigRef.md

```diff
@@ -390,6 +390,12 @@ The various cache directory locations are all under `/{tds.content.root.path}/th
 
 We recommend that you use these defaults, by not specifying them in the `threddsConfig.xml` file.
 If you need to move the cache location, move all of them by using a symbolic file link in `${tds.content.root.path}/thredds/`.
+At Unidata, we move the entire content directory by creating a symbolic link:
```

Comment:
@oxelson - I was synching the various ThreddsConfigRef docs, and this line was in the admin guide but not the user guide (which has been updated quite a bit). Should we keep this addition?

Reply:
Sure, no harm in it.